packages/demos/src/commonjs-module-loader/integrator.node.tsx

```diff
@@ -1,5 +1,5 @@
 import {FeatureAppManager, FeatureServiceRegistry} from '@feature-hub/core';
-import {loadCommonJsModule} from '@feature-hub/module-loader/lib/node';
+import {loadCommonJsModule} from '@feature-hub/module-loader/lib/cjs/node';
```

Comment:
do we also refer to cjs in the esm build?

Reply:
I'm not sure I understand. This import is in a demo which doesn't get built to esms.